app/front/scripts/components/babbage-geoview/render.js

```diff
@@ -155,6 +160,38 @@ function renderInfoCard(layer, options) {
   };
 }
 
+function renderLegend(layer, options) {
```

Comment:
No, it's not used. Also found the same in `renderInfoCard()` function. Will remove both.